ios/Classes/Error/BleError.m

```diff
@@ -1,5 +1,5 @@
 #import "BleError.h"
-#import "JSONStringifier.h"
+#import "JSONStringifier1.h"
```

Comment:
BlemulatorJSONStringifier?

Reply:
Changed according to the review suggestion.